docker/Dockerfile-dhvirtualenv

```diff
@@ -14,32 +14,29 @@ ARG distro=""
 
 ###
 ### Stage 0: build a dh-virtualenv
+### Based on https://github.com/spotify/dh-virtualenv/blob/master/Dockerfile
 ###
+
 FROM ${distro} as builder
 
-RUN apt-get update -qq -o Acquire::Languages=none
-RUN env DEBIAN_FRONTEND=noninteractive apt-get install \
-        -yqq --no-install-recommends \
-        build-essential \
-        ca-certificates \
-        devscripts \
-        equivs \
-        wget
+RUN apt-get update -qq -o Acquire::Languages=none \
+    && env DEBIAN_FRONTEND=noninteractive apt-get install \
+        -yqq --no-install-recommends -o Dpkg::Options::=--force-unsafe-io \
+        build-essential debhelper devscripts equivs lsb-release \
+        python3-setuptools \
+        python3-sphinx python3-mock dh-exec dh-python python3-sphinx-rtd-theme \
```

Comment:
are we basically hardcoding the build-deps of dh-virtualenv here? I'm not convinced that is an improvement, since it means we now need to keep it in sync with dh-virtualenv.

Reply:
Yep, but 1) upstream updates are rather rare and 2) it's far easier to track down discrepancies this way (the `buster` builds failing took me a while to sort out as a result).

It also avoids another call out to the network for `mk-build-deps` and lets us get all of that out of the way (and cached) up front.

Reply:
I'm surprised that the old incantation didn't just work. 

anyway, fair enough if you want to go this way. It just always feels wrong to me when you have to manually keep a list of packages in sync.